python/rapidsmpf/CMakeLists.txt

```diff
@@ -19,6 +19,16 @@ project(
   LANGUAGES CXX CUDA
 )
 
+# Add global AddressSanitizer flags only if explicitly requested
+option(RAPIDSMPF_PYTHON_ASAN "Enable AddressSanitizer for Python build" OFF)
+add_library(maybe_asan INTERFACE)
+target_compile_options(
```

Comment:
You can simplify this by using `$<COMPILE_LANGUAGE:C,CXX>`  instead

Reply:
According to [this](https://cmake.org/cmake/help/latest/manual/cmake-generator-expressions.7.html#genex:COMPILE_LANGUAGE:languages), my understanding was correct.